### PR TITLE
feat: add MLL_PALETTE constant with 12 primary colors plus black and white

### PR DESCRIPTION
**Pull request by SimBig**

### CODE REVIEW

File: src/theme.ts [hunk 1]
```diff
@@ -1,14 +1,41 @@
...
 import { useContext } from 'react';
 import { ThemeContext } from 'styled-components';
 
-/**
- * We omit special purpose colors:
- * - 'white': Brightest possible
- * - 'black': Darkest possible
- * - 'transparent': No color
- */
+export const MLL_PALETTE = {
```
Comment (spawnia) [inline src/theme.ts:21]:
```suggestion
export const PALETTE = {
```

File: src/theme.ts [hunk 2]
```diff
@@ -1,14 +1,41 @@
...
+  blue: blue.primary,
+  geekblue: geekblue.primary,
+  purple: purple.primary,
+  magenta: magenta.primary,
+  grey: grey.primary,
+  white: '#ffffff',
+  black: '#000000',
+};
+
 export const MLL_THEME = {
```
Comment (spawnia) [inline src/theme.ts:39]:
```suggestion
export const THEME = {
```

File: src/theme.ts [hunk 3]
```diff
@@ -1,14 +1,41 @@
...
 import { ThemeContext } from 'styled-components';
 
-/**
- * We omit special purpose colors:
- * - 'white': Brightest possible
- * - 'black': Darkest possible
- * - 'transparent': No color
- */
+export const MLL_PALETTE = {
+  red: red.primary,
```
Comment (spawnia) [inline src/theme.ts:22]: This should be MLL red - `#ff000`

File: src/theme.ts [hunk 4]
```diff
@@ -1,14 +1,41 @@
...
+  geekblue: geekblue.primary,
+  purple: purple.primary,
+  magenta: magenta.primary,
+  grey: grey.primary,
+  white: '#ffffff',
+  black: '#000000',
+};
+
 export const MLL_THEME = {
   borderColor: '#50a0d0',
```
Comment (spawnia) [inline src/theme.ts:40]: We should define those in terms of the palette. This has the benefit of forcing us to either:
- increase the consistency of the theme
- discover missing colors and adding them to the palette

I would err on the side of fewer colors and a small palette.

**Review by spawnia — CHANGES_REQUESTED** [review]

File: src/theme.ts (hunk 2 above)
Comment (spawnia) [reply src/theme.ts:42]: yes

File: src/theme.ts (hunk 3 above)
Comment (spawnia) [reply src/theme.ts]: I think having two slightly different shades of red on the same page looks bad regardless of their purpose.

File: src/theme.ts (hunk 4 above)
Comment (SimBig) [reply src/theme.ts]: I was able to merge some colors and I've defined grey tints and specific colors for the remaining colors.
@spawnia let me know if and how you would handle the other colors.

File: src/theme.ts (hunk 2 above)
Comment (SimBig) [reply src/theme.ts]: I've added a deprecated copy of THEME so that we have no stress to replace the old usages..

File: src/Plate/index.tsx [hunk 5]
```diff
@@ -210,16 +210,16 @@ function Well(props: { position: number; well?: PlateWell }) {
     <span
       style={{
         backgroundColor: props.well?.selected
-          ? MLL_THEME.warningColor
-          : MLL_THEME.tableBorderColor,
+          ? THEME.warningColor
+          : THEME.tableBorderColor,
```
Comment (spawnia) [inline src/Plate/index.tsx:214]: Redefine in terms of the palette? @SimBig please check other usages where the theme colors are used and the semantics don't line up.

File: src/theme.ts [hunk 6]
```diff
@@ -1,33 +1,70 @@
...
+  gray1: '#fafafa',
+  gray2: '#f0f0f0',
+  gray3: '#e6e6e6',
+  gray4: '#d9d9d9',
+  gray5: '#c0c0c0',
+  gray6: '#8c8c8c',
+  gray7: '#595959',
+  gray8: '#434343',
+  gray9: '#262626',
+  black: '#000000', // darkest possible
```
Comment (SimBig) [inline src/theme.ts:27]: @spawnia I've added a gray scale that should suit our needs and partly mapped the current hex values to it. 
The remaining deprecated values have been extracted to a new const and should be replaced in limes-frontend with the new values available

**Review by spawnia — APPROVED** [review]

File: src/theme.ts (hunk 6 above)
Comment (spawnia) [reply src/theme.ts:27]: nice!

**Pull request merged**